client/starwhale/consts/__init__.py

```diff
@@ -1,5 +1,7 @@
 import pathlib
+import threading
 
+thread_local = threading.local()
```

Comment:
use `threading.local()` directly?

if main thread called `consts.thread_local`, the other threads import the unexpected `thread_local` ?